code/utils/io/FileReader.py

```diff
@@ -0,0 +1,30 @@
+import logging as LOG
+from customexceptions.ReadabilityErrors import IOError
+
+
+class FileReader:
+    def __init__(self):
+        """
+        File Reader init
+        """
+        LOG.debug(f'{__name__} init')
+
+    def read(self, filename: str) -> str:
+        """
+        Read the contents of a file
+        :param filename: The name of the file to read
+        :return: The file contents if read is successful. Else None
+        """
+        LOG.debug(f'Reading file {filename}')
+        try:
+            with open(filename, 'r', encoding='utf-8') as reader:
+                tempString = ""
+                for line in reader:
+                    tempString += line + '\n'
```

Comment:
Wont the line already have a newline char?

Reply:
Yes, it will have. This code would add a blank line in between all the lines in the file.

Reply:
Silly me. I thought I had fixed it. Will do